docs/docs/configuration/alpha_config.md

```diff
@@ -67,6 +67,20 @@ the new config.
 oauth2-proxy --alpha-config ./path/to/new/config.yaml --config ./path/to/existing/config.cfg
 ```
 
+## Using ENV variables in the alpha configuration
+
+The alpha package supports the use of environmental variables in place of yaml keys, allowing sensitive values to be pulled from somewhere other than the yaml file.
```

Comment:
Please replace all occurrences of `environmental variables` with `environment variables`

Reply:
done + updated link to new library documentation